app/controllers/DetailsUpdatedController.scala

```diff
@@ -31,7 +33,7 @@ class DetailsUpdatedController @Inject() (
 ) extends FrontendBaseController
     with I18nSupport {
 
-  def onPageLoad: Action[AnyContent] = standardActionSets.identifiedUserWithData() {
+  def onPageLoad: Action[AnyContent] = Action { // standardActionSets.identifiedUserWithData() {
```

Comment:
Why is there no action on this? Do we not want to at least check the user is authorised / logged in or _something_?